src/uct/tcp/tcpcm/configure.m4

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+AC_CONFIG_FILES([src/uct/tcp/tcpcm/Makefile])
```

Comment:
Since it doesn't check any dependencies, we don't need configure.m4 for UCT/TCP/CM code. Follow the same approach as for UCT/TCP

Reply:
I thought I removed this file. Removing now.